src/domains/job/search/detail/page/JobSeekDetailPage.jsx

```
@@ -44,12 +48,24 @@ const JobSeekDetailPage = () => {
     }
   }
 
+  const handleScrapClick = async () => {
+    try {
+      await toggleScrap(id, 'JOB_SEEKING')
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)